tests/integration/components/frost-button-test-.js

```diff
@@ -2,6 +2,7 @@ import {expect} from 'chai'
 import {describeComponent, it} from 'ember-mocha'
 import hbs from 'htmlbars-inline-precompile'
 import {beforeEach} from 'mocha'
+import {$hook, initialize} from 'ember-hook'
 
 describeComponent(
   'frost-button',
```

Comment:
for some reason frost button integration tests are not running. all other component tests are running. Thats why this one is sparse for now.